integration-tests/common/test_common.go

```diff
@@ -136,7 +138,7 @@ func (m *OCRv2TestState) DeployCluster() {
 	// When running soak we need to use K8S
 	if *m.Common.TestConfig.Common.InsideK8s {
 		m.DeployEnv()
-
+		m.StartGppWithoutNetwork()
```

Comment:
nit: a better name is `StartGppDefaultNetwork` b/c all containers have a network